mqttwarn/services/desktopnotify.py

```
@@ -30,27 +40,36 @@ def plugin(srv: Service, item: ProcessorItem):
+    if message is None:
+        raise ValueError("Message is required")
+    if is_json(message):
+       parsed = json.loads(message)
+       data = parsed if isinstance(parsed, dict) else {
+         "title": item.get("title", item.topic),
+         "message": parsed,
+       }
+    else:       
+       try:
+          msg = t.cast(t.Dict, message).get('message')
+       except:
```

Ruff (0.15.20)

[error] Do not use bare `except`

(E722)